libpod/container_internal.go

```diff
@@ -1199,25 +1195,39 @@ func (c *Container) setupOCIHooks(ctx context.Context, config *spec.Spec) (exten
 		}
 	}
 
-	allHooks := make(map[string][]spec.Hook)
-	for _, hDir := range c.runtime.config.HooksDir {
-		manager, err := hooks.New(ctx, []string{hDir}, []string{"poststop"}, lang)
-		if err != nil {
-			if c.runtime.config.HooksDirNotExistFatal || !os.IsNotExist(err) {
-				return nil, err
-			}
-			logrus.Warnf("failed to load hooks: %q", err)
+	if c.runtime.config.HooksDir == nil {
+		if rootless.IsRootless() {
```

Comment:
should we just change the default for rootless containers to point somewhere in the user directory?  So that we don't have to disable hooks completely

Reply:
These root defaults are deprecated in favor of explicitly-configured hook directories.  Rootless has no current defaults, so nothing to deprecate. Everyone (rootless or not) can now explicitly configure hook dirs, but I don't see a point to giving rootless a deprecated-from-birth default.

Reply:
yes makes sense, thanks for the clarification